src/nuclio/common/components/edit-item/edit-item.component.js

```diff
@@ -248,6 +252,24 @@
             $scope.$on('deploy-function-version', onFunctionDeploy);
         }
 
+        function updateNameValidationRules() {
+            if (ctrl.isTriggerType() && !lodash.isEmpty(ctrl.selectedClass) && !ctrl.isHttpTrigger()) {
+                if (!lodash.find(ctrl.validationRules.itemName, ['name', 'mustNotBe'])) {
+                    ctrl.validationRules.itemName.push({
+                        name: 'mustNotBe',
+                        label: $i18next.t('common:MUST_NOT_BE', {lng: lng}) + ': default-http',
+                        pattern: function (value) {
+                            return value !== 'default-http'
+                        }
+                    })
+                }
+            } else {
+                lodash.remove(ctrl.validationRules.itemName, function (rule) {
+                    return rule.name === 'mustNotBe'
+                })
```

Comment:
You can use the shorter form of `lodash.remove`'s parameters:
```suggestion
                lodash.remove(ctrl.validationRules.itemName, ['name', 'mustNotBe']);
```